GitUI/GitUICommands.cs

```diff
@@ -1487,6 +1487,14 @@ private bool RunCommandBasedOnArgument(IReadOnlyList<string> args, IReadOnlyDict
                         return StartCloneDialog(null, args[1].Replace("github-mac://openRepo/", ""), true);
                     }
 
+                    // User supplied a path. Open the repository if its a valid path
+                    string dir = !string.IsNullOrWhiteSpace(args[1]) && File.Exists(args[1]) ? Path.GetDirectoryName(args[1]) : args[1];
+                    if (args.Count == 2 && Directory.Exists(dir))
+                    {
+                        LaunchBrowse(dir);
```

Comment:
Exception if no argument...
untested

```suggestion
                    string dir = args.Count != 2 
                        ? ""
                        :  File.Exists(args[1]) ? Path.GetDirectoryName(args[1]) : args[1];
                    if (Directory.Exists(dir))
                    {
                        LaunchBrowse(dir);
```

Reply:
I don't see any exception if I am passing no args. Could you please give evidence?

Since we are `string.IsNullOrWhiteSpace(args[1])` invalid cases will be ignored.